SPECS/harfbuzz/harfbuzz.spec

```diff
@@ -76,6 +77,9 @@ find . -type f -name "*.py" -exec sed -i'' -e '1 s|^#!\s*/usr/bin/env\s\+python3
 %{_libdir}/libharfbuzz-icu.so.*
 
 %changelog
+* Fri Feb 17 2023 corvus-callidus <corvus-callidus <108946721+corvus-callidus@users.noreply.github.com> - 3.4.0-3
```

Comment:
Related to something I've noticed in #4881: please use your full name and work e-mail in these changelog entries. You can take the entry below as an example.